jupyterlab_h5web/handlers.py

```diff
@@ -25,12 +25,12 @@ def get(self):
         with h5py.File(as_absolute_path(self.base_dir, Path(file_path)), "r") as h5file:
             content = self.get_content(h5file, path)
 
-        chunks, headers = encode(content, format_arg)
+        response = encode(content, format_arg)
```

Comment:
Adapt to changes in [h5grove 0.0.14](https://github.com/silx-kit/h5grove/releases/tag/v0.0.14)